packages/fxa-settings/src/components/Settings/PageRecoveryPhoneRemove/en.ftl

```diff
@@ -0,0 +1,11 @@
+## PageRecoveryPhoneRemove
+## Users reach this page from account settings when they want to remove a backup phone number.
+
+recovery-phone-remove-header = Remove recovery phone number
+# Variables:
+#   $formattedFullPhoneNumber (String) - the user's full phone number
+settings-recovery-phone-remove-info = This will remove <strong>{ $formattedFullPhoneNumber }</strong> as your backup recovery phone.
```

Comment:
Is it `recovery phone` or `backup recovery phone` now? (I remember reviewing a patch where strings removed the `backup`.)

Reply:
It should be `recovery phone`. Good catch!

Reply:
Ahh yes nice catch. Sorry about that, this was recently changed.